src/components/MVP/CovidSiteAlert/CovidSiteAlert.test.tsx

```diff
@@ -3,12 +3,12 @@
  */
 import React from 'react'
 import { render, screen } from '@testing-library/react'
-
+import { axe, toHaveNoViolations } from 'jest-axe'
 import CovidSiteAlert from './CovidSiteAlert'
 
+expect.extend(toHaveNoViolations)
```

Comment:
I think we can call `expect.extend` in the jest setup file to extend it globally so we don't need to add this to each test file!

Reply:
ahh i was trying to get that to work yesterday but couldn't access `expect` in the file

Reply:
yeah, eslint gets mad but I think the tests will still run. I think there is some way of telling eslint that `expect` is a global but I am not sure

Reply:
oh -- i see. it actually _does_ work, even though eslint is mad that expect is undefined.